README.md

```diff
@@ -50,12 +50,19 @@ Hints:
 
 ## Running the tests
 
-Just run `./gradlew test`.
+Build a docker image using the first part of the Dockerfile.
+```shell
+docker build . --target tester --tag "backend:tester"
+```
+
+Then run the built image.
+```shell
+docker run --name backend-tester --rm "backend:tester"
+```
+
+### Jasmine
 
-Under Linux you _might_ also need to set `OPENSSL_CONF=/dev/null` environment variable.
-Without that PhantomJS might run into trouble.
-Also, make sure you have `chromedriver` on your path.
-Those two problems will be solved once we Dockerize test environment.
+The frontend code is tested using a Javascript test framework called Jasmine. The package.json file defines its configuration.
```

Comment:
Probably "too fast" merged changes from master. Those lines are important, without those I, and maybe others, will not be able to execute the tests locally.

Reply:
Are you able to run it today?

Reply:
Yes, but only when I do exactly those two things mentioned in README. So:
`PATH="/snap/chromium/current/usr/lib/chromium-browser":"$PATH" OPENSSL_CONF=/dev/null ./gradlew clean test`